package.json

```diff
@@ -3,8 +3,10 @@
   "version": "3.6.0",
   "description": "map",
   "scripts": {
-    "build": "cross-env NODE_ENV=production webpack --mode=production",
-    "start": "cross-env NODE_ENV=development webpack serve",
+  	"build": "echo use 'npm run prodbuild' or 'npm run devbuild'",
+    "prodbuild": "cross-env NODE_ENV=production webpack --mode=production",
```

Comment:
The instructions in `documents/deployment.md` should also be updated to match this change.

Reply:
fixed ✔️